client.go

```diff
@@ -40,3 +42,26 @@ func (s *Service) NewFirebaseClient() (firebase.Client, error) {
 	}
 	return firebase.NewClient(baseUrl, authToken, nil), nil
 }
+
+
+func (s *Service) NewRedisClient() (redis.Conn, *redis.PubSubConn, error) {
+	redisTcpAddr, err := s.Env.GetString("REDIS_PORT_6379_TCP_ADDR")
```

Comment:
You're not doing anything with these errors. They should be `_` or returned if they occur.